lib/src/git_backend.rs

```diff
@@ -1022,6 +1024,18 @@ impl Backend for GitBackend {
         Ok(SymlinkId::new(oid.as_bytes().to_vec()))
     }
 
+    fn read_copy(&self, _id: &CopyId) -> BackendResult<CopyHistory> {
+        unimplemented!();
```

Comment:
nit: maybe better to return `BackendError::Unsupported` if we aren't going to implement these functions anytime soon.